examples/webpack5/BUILD.bazel

```diff
@@ -0,0 +1,13 @@
+load("@npm_deps//webpack:index.bzl", "webpack")
+
+webpack(
+    name = "serve",
+    args = [
+        "serve",
+    ],
+    data = glob(["src/**"]) + [
```

Comment:
Dont you want "ibazel_notify_changes", here so that ibazel won't restart the devserver process?

Reply:
Yes I need to add that. The one issue I hit on Friday was that `ibazel` for some reason invokes the process in a way that doesn't allow it to receive fs events, thus `webpack-dev-server` won't respond to files being modified. I haven't dug into why this is the case yet. But the example above (though it doesn't provide the rebuild benefit that `ibazel` would provide, it does allow the dev server to respond to file changes in the run tree.